### ISSUE TITLE
"Accessibility Text" method in GoogleAssistant CardImage

### ISSUE THREAD

**Issue by davidmarinangeli** [dialogflow/dialogflow-java-client#45]
Hello guys, I'm developing an Actions On Google app and, developing a list of CardItem I've realised a little bug of your library.

In the CardImage method is missing an important tag called "accessibility_text" that gives the following error on Google Assistant: 

> MalformedResponse
expected_inputs[0].possible_intents[0].input_value_data.option_value_spec.list_select.items[0].image: 'accessibility_text' is required.

I would like to help you and add it by myself: could i fork and pull request it?

**Comment by davidmarinangeli** [comment]
Hi! I totally forgot this issue but I can give you some further info that I have learnt: the Malformed Response issue happens because of the lack of "accessibility_text" tag in the JSON. 

Actions on Google console gives me this error (...but I think that's a warning since the app works anyway): 
> MalformedResponse
>expected_inputs[0].possible_intents[0].input_value_data.option_value_spec.carousel_select.items[0].image: 'accessibility_text' is required.

Furthermore in the documentations of Rich Responses 

```
"basicCard": {
                                "title": "Math & prime numbers",
                                "formattedText": "42 is an even composite number. It\n    is composed of three distinct prime numbers multiplied together. It\n    has a total of eight divisors. 42 is an abundant number, because the\n    sum of its proper divisors 54 is greater than itself. To count from\n    1 to 42 would take you about twenty-one…",
                                "image": {
                                    "url": "https://example.google.com/42.png",
                                    **"accessibilityText": "Image alternate text"**
                                },
```

as you can see, in the object "Image" there is a field accessibilityText that these APIs don't have. 

I really would like to see a big growth of the Java API of Dialogflow since I, and other devs, don't love Node.JS
